symengine/tests/cwrapper/test_cwrapper.c

```diff
@@ -83,11 +83,31 @@ void test_basic() {
     basic_free_heap(y);
 }
 
+void test_complex() {
+    basic e;
+    basic f;
+    char* s;
+    basic_init(e);
+    basic_init(f);
```

Comment:
What error are you getting? CMake in the log you posted seems to succeed .

Sent from my mobile phone.
On Jul 28, 2015 11:24 AM, "Shivam Vats" <notifications@github.com> wrote:

> In symengine/tests/cwrapper/test_cwrapper.c
> <https://github.com/sympy/symengine/pull/562#discussion_r35674572>:
>
> > @@ -83,11 +83,31 @@ void test_basic() {
> >      basic_free_heap(y);
> >  }
> >
> > +void test_complex() {
> > +    basic e;
> > +    basic f;
> > +    char* s;
> > +    basic_init(e);
> > +    basic_init(f);
>
> You are right. I am sorry, I shouldn't have overlooked this.
> I couldn't use Debug mode as my master throws errors
> <https://gitter.im/sympy/symengine> while building in Debug mode. I am
> still trying to figure this out.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sympy/symengine/pull/562/files#r35674572>.
>